clang/test/SemaCXX/static-assert-cxx20.cpp

```diff
@@ -0,0 +1,5 @@
+// RUN: %clang_cc1 -std=c++2a -triple=x86_64-linux -fsyntax-only %s -verify
```

Comment:
You can add another C++20 mode run to clang/test/SemaCXX/static-assert-cxx26.cpp, instead of adding a new file.

Reply:
The file refers to cxx26 —is it ok to add tests related to C++20 here?"

Reply:
I believe so, yes. See the example: clang/test/SemaCXX/constant-expression-cxx2b.cpp

Reply:
Also, you might want to wrap the case with a namespace referring to the issue number.

Furthermore, I think the test should be triaged as a Parser one. Can you please move the test to that directory?